src/Symfony/Bridge/Doctrine/Listener/DoctrineConnectionListener.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Listener;
+
+use Doctrine\DBAL\Connection;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class DoctrineConnectionListener implements EventSubscriberInterface
+{
+    private array $lastChecked = [];
+
+    public function __construct(private readonly ContainerInterface $container, private readonly array $timingByConnection)
+    {
+        foreach($this->timingByConnection as $params) {
+            $this->lastChecked[$params['name'] = 0];
```

Comment:
Shouldn't the line look like this?

```suggestion
            $this->lastChecked[$params['name']] = 0;
```

Reply:
oh indeed!